examples/unionjack.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import colorsys
+import time
+import blinkt
+
+Red   = (255,0,0)
+White = (127,127,127)
+Blue  = (0,0,255)
+
+Pattern = \
```

Comment:
Should probably be lowercase `p` since `Pattern` would indicate a Python class.

Reply:
Done